README.md

```diff
@@ -1,2 +1,30 @@
 # hpccarpentry-mini-intro
-material for our course on hpccarpentry
+material for our course on hpccarpentry.
+
+There were max 16 people last week so we can probably expect the same. This would mean a max of 3 breakout rooms.
+
+# Agenda
+## What is HPC Carpentry? (20 mins, including discussion)
+* Introduction to HPC Carpentry and what it's origins and goals are
+* What HPC Carpentry is *not*
+## (Inspired ideas from Peter go here)
+
+## Lesson development for Centres of Excellence
+FocusCoE has funded the creation of a template lesson on how to run a community code **at scale**. The idea being that such a lesson might be useful to CoEs who typically have a marquee scalable application. We chose LAMMPS as the test case because it is used heavily by the E-CAM community and development on top of LAMMPS is very common. 
+
+### Overview of lesson design (20 minutes)
+* What are the learner profiles?
```

Comment:
I just realised that learner profiles don't seem to be recorded alongside lessons. Even for Software Carpentry, the profiles cover the full set of lessons, they aren't individually connected to the lessons. This feels like a shortcoming to me giving how we are advised to approach lesson design.

Reply:
We should/can improve on that.